README.md

```diff
@@ -99,3 +99,20 @@ doc_gen.generate_documentation(
 	write_documentation_to_yaml=False
 )
 ```
+
+#### How it works
+
+The Chatbot is based on the concept of Retrieval Augmented Generation and basically works as follows:
+
+- When you call the `chatbot.load_models()` method, the bot scans all the folders in the locations specified by you for dbt YML files.
+- It then converts all the models into a text description, which are stored as embeddings in a vector database. The bot currently only supports [ChromaDB](https://www.trychroma.com/) as a vector db, which is persisted in a file on your local machine.
+- When you ask a query, it fetches 3 models whose description is found to be the most relevant for your query.
+- These models are then fed into ChatGPT as a prompt, along with some basic instructions and your question.
+- The response is returned to you as a string.
+
+## Contributors
```

Comment:
I think we can call this section "Partners" and only add a reference to the OSDC. No point in repeating my name as well, since it's also already present in the contributors list.

Reply:
Sure Pragun, will push the change right now.